tools/style.sh

```diff
@@ -25,7 +25,7 @@ black . --color --diff --check
 echo -e "::endgroup::"
 
 echo -e "\n::group:: ===> check mypy"
-mypy . --no-site-packages
+mypy . --no-site-packages --ignore-missing-imports
```

Comment:
I don’t think we needed this change because it should read the mypy config

Reply:
I was not sure. I removed this flag since I also added it in setup.cfg.